resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/ExecutorPodsAllocator.scala

```diff
@@ -47,6 +49,16 @@ private[spark] class ExecutorPodsAllocator(
 
   private val podCreationTimeout = math.max(podAllocationDelay * 5, 60000)
 
+  // maxNumExecutorFailures must be less than Int.MaxValue
+  private lazy val maxNumExecutorFailures = conf.get(KUBERNETES_MAX_EXECUTOR_FAILURES)
+    .getOrElse {
+      math.max(3, if (totalExpectedExecutors.get() > Int.MaxValue / 2) {
+        Int.MaxValue
+      } else {
+        (2 * totalExpectedExecutors.get())
```

Comment:
`(2 * totalExpectedExecutors.get())` -> `2 * totalExpectedExecutors.get()`?

Reply:
Yes, I'll change that